### PR TITLE
Some docs on EventMachine adapters.

### PR DESCRIPTION
**Pull request by damau**
## Description
Hey.  I spent some time digging through the EM adapters so I thought I would document how to use the parallel bits.

- Tests - I tested the docs boot up.

## Additional Notes
I tried to manage expectations and inform a little on the adapter page.  It does behave differently, though obviously for good reason.  It's also why I decided not to put parallelization onto the README or front pages.

Also big thanks for all the maintainers.  Long time user of Faraday with some time off.

### CODE REVIEW

File: docs/adapters/em-http.md [hunk 1]
```diff
@@ -7,20 +7,65 @@ top_name: Adapters
...
+
+urls = Array.new(5) { 'http://127.0.0.1:3000' }
+
+conn = Faraday::Connection.new do |builder|
+  builder.adapter :em_http
+end
+
+begin
+  conn.in_parallel do
+    puts "Parrallel manager: #{conn.parallel_manager}"
```
Comment (olleolleolle) [inline docs/adapters/em-http.md:50]: A typo, just one R in parallel.

File: docs/adapters/em-synchrony.md [hunk 2]
```diff
@@ -7,15 +7,65 @@ top_name: Adapters
...
+
+urls = Array.new(5) { 'http://127.0.0.1:3000' }
+
+conn = Faraday::Connection.new do |builder|
+  builder.adapter :em_synchrony
+end
+
+begin
+  conn.in_parallel do
+    puts "Parrallel manager: #{conn.parallel_manager}"
```
Comment (olleolleolle) [inline docs/adapters/em-synchrony.md:56]: Same here

**Review by olleolleolle — APPROVED** [review]
Thanks for improving the docs for these adapters!

I read the change, and only marked a single-letter typo. This is super.

**Review by iMacTia — APPROVED** [review]
Nicely done and great improvement to the documentation!
We definitely lack knowledge around the EM area and that affects both adapters code and documentation, unfortunately.

We very much welcome the help, and if your "little time off" would allow then we also welcome new maintainers for the adapter that are being moved to external gems 😄
If interested you can read more on [this comment](https://github.com/lostisland/faraday/issues/953#issuecomment-752938926)

**Comment by damau** [comment]
Ta, I'll take a peak.

**Comment by iMacTia** [comment]
@damau I'm so sorry I have no idea why this wasn't merged yet and I just noticed it!
Thanks again for the this contribution, I'm sure it will help many in future ❤️

**Comment by damau** [comment]
Really late reply but no worries!!

**Pull request merged**